package.json

```
@@ -0,0 +1,40 @@
+{
+    "name": "ocsp",
+    "description": "OpenSSL ocsp bindings",
+    "version": "1.0.0",
+    "gypfile": true,
+    "main": "dist/index.js",
+    "scripts": {
+        "install": "node-gyp rebuild",
+        "lint": "tslint -t codeFrame 'index.ts' 'test/**/*.ts' && prettier-check 'index.ts' 'test/**'",
+        "package": "yarn lint && yarn test && tsc -p .",
+        "test": "jest"
+    },
```

Nit: I'm seeing that you're committing the `dist` folder, thus having to run the `package` script manually before submitting a PR or a commit.

Maybe you could use the `prepare` script that will do this for you at install time.
In combinaison with the `directories` entry in the `package.json`, you'll have a very minimal footprint where it gets installed.

With this, you'll also have to keep the `dist` folder in `.gitignore`.

```suggestion
        "prepare": "yarn package"
    },
    "directories": {
        "lib": "dist"
    },
```

I didn't test it in this context, if you like it I'd advise to test it and see that everything's working as expected.

---

It's something I did not know how to do properly. Seems it's the right time to fix it.

---

Last time I tried something like this, I failed to make it work with install from `git` package :/. But it was a long time ago, let's give it a try and hope it works because I agree it's the right way to do!

---

Do not invest too much time in it if you can't make it work, 
it seems to possibly be buggy with `yarn` and git packages.
https://github.com/yarnpkg/yarn/issues/5235

Can you confirm it would work @arcanis?

---

Made a quick test, the prepare/prepack scripts don't seem to be called - although install/postinstall are, but that's not what we want 🤔

Depending on the urgency I can work on a fix for Yarn and get it released in a day or two.

---

No urgency! As always it'd be nicer to do the clean way but we can totally live with `dist` committed for now.